bitnami/jaeger/templates/query/deployment.yaml

```diff
@@ -0,0 +1,114 @@
+      {{- if .Values.affinity }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.podAffinityPreset "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.podAntiAffinityPreset "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.nodeAffinityPreset.type "key" .Values.nodeAffinityPreset.key "values" .Values.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.topologySpreadConstraints "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: jaeger-query
+          image: {{ include "jaeger.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.query.command "context" $) | nindent 12 }}
+          {{- if .Values.query.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.query.args "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.query.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.query.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.image.debug | quote }}
+            {{- if .Values.query.extraEnvVars }}
+              {{- include "common.tplvalues.render" (dict "value" .Values.query.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.query.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.query.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.query.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.query.extraEnvVarsSecret }}
+            {{- end }}
+          {{- if .Values.query.customStartupProbe }}
```

Comment:
I'm missing the default probes. Even though you cannot still check it properly, you can try checking the upstream chart code or even the source code for a probe.

Reply:
done [[388bdf9](https://github.com/bitnami/charts/pull/13480/commits/388bdf906475ce4f33a564cb4f563d2a5fa039be), [6bd59a9](https://github.com/bitnami/charts/pull/13480/commits/6bd59a9e8cee4d4d35e641e36232a47865f1b218), [ef55bd4](https://github.com/bitnami/charts/pull/13480/commits/ef55bd46f1133ca30eca6697f6358d0b8c6658da)]

Reply:
Fixed [f6bb327](https://github.com/bitnami/charts/pull/13480/commits/f6bb3273576ee3aa4247991bfdc892dd2bf7aac1)